ArduPlane/altitude.cpp

```diff
@@ -358,26 +358,39 @@ int32_t Plane::calc_altitude_error_cm(void)
 }
 
 /*
-  check for FBWB_min_altitude_cm violation
- */
-void Plane::check_fbwb_minimum_altitude(void)
-{
-    if (g.FBWB_min_altitude_cm == 0) {
+  check for min and max altitude: FBWB_min_altitude_cm and fbwb_max_altitude_m violation
+*/
+void Plane::check_fbwb_altitude(void)
+{   int16_t min_difference = 1000 ; //  there should be a minimum difference between max and min altitude (in cms)
+    bool should_check_max = ((g.fbwb_max_altitude_m * 100 > g.FBWB_min_altitude_cm + min_difference) && g.fbwb_max_altitude_m > 0); //larger than alt min AND fbwb_max_altitude_m > 0
+    bool should_check_min = (g.FBWB_min_altitude_cm != 0);
```

Comment:
```suggestion
{  
    const int16_t min_difference_cm = 1000; // there should be a minimum difference between max and min altitude (in cms)
    bool should_check_max = (g.fbwb_max_altitude_m * 100 > g.FBWB_min_altitude_cm + min_difference_cm) && g.fbwb_max_altitude_m > 0; // larger than alt min AND fbwb_max_altitude_m > 0
    bool should_check_min = g.FBWB_min_altitude_cm != 0;
```
Just some formatting changes